ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml

```diff
@@ -59,7 +59,7 @@ ModalPopup {
             return sendingError.open()
         }
 
-        sendingSuccess.text = qsTr("Transaction sent to the blockchain. You can watch the progress on Etherscan: %2/%1").arg(response.result).arg(walletModel.etherscanLink)
+        sendingSuccess.text = qsTr("Transaction sent to the blockchain. You can watch the progress on Etherscan: %2%1").arg(response.result).arg(walletModel.etherscanLink)
```

Comment:
Do we still need to show this? I think it was meant to be used as a temporary solution while we implemented the toast messages.